src/main/java/first/controllers/StudentController.java

```diff
@@ -1,257 +1,57 @@
 package first.controllers;
 
+import first.dto.StudentDto;
 import first.model.Student;
 import first.repository.StudentRepository;
-import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.*;
 
-import java.sql.SQLException;
-import java.util.Iterator;
-import java.util.LinkedList;
-import java.util.List;
-
-@Controller
 @RestController
-public class StudentController {
+public class StudentController implements StudentControllerInter {
     @RequestMapping(value = "/form", method = RequestMethod.GET)
     @ResponseBody
     String getStudentForm() {
-        String FORM = "<html>\n" +
-                " <head>\n" +
-                " </head>\n" +
-                " <meta http-equiv=\"Content-Type\"  content=\"text/html; charset=utf-8\">\n" +
-                " <body> \n" +
-                " <form action=\"/viewstud\" method=\"get\" >\n" +
-                "  <p>Имя:<br>\n" +
-                "   <input type=\"text\" name=\"name\" size=\"40\">\n" +
-                "  </p>\n" +
-                "  <p>Фамилия:<br>\n" +
-                "   <input type=\"text\" name=\"surname\" size=\"40\">\n" +
-                "  </p>\n" +
-                "  <p>Возраст:<br>\n" +
-                "   <input type=\"number\" name=\"age\" size=\"2\">\n" +
-                "  </p>\n" +
-                "  <p>Оценка:<br>\n" +
-                "   <input type=\"number\" name=\"mark\" size=\"2\">\n" +
-                "  </p>\n" +
-                "  <p>Курс:<br>\n" +
-                "   <input type=\"number\" name=\"course\" size=\"1\">\n" +
-                "  </p>\n" +
-                "   <p><input type=\"submit\" name=\"button\" value=\" Добавить \"></p>\n" +
-                "  </form>\n" +
-                " </body>\n" +
-                "</html>";
-        return FORM;
-
+        StudentDto studAdd = new StudentDto();
+        return studAdd.add();
     }
-
     @RequestMapping(value = "/viewstud", method = RequestMethod.GET)
     @ResponseBody
     public String viewstud(@RequestParam(value = "name") String name, @RequestParam(value = "surname") String surname, @RequestParam(value = "age") Integer age, @RequestParam(value = "mark") Integer mark, @RequestParam(value = "course") Integer course, @RequestParam(value = "button") String button) {
-
         System.out.println( "Данные зашли" );
         Student studentReady = new Student( name, surname, age, mark, course );
         System.out.println( "Студент создан" );
         StudentRepository stud = new StudentRepository();
```

Comment:
Could you please move this instance to class variable? 
Do we really need every time create new StudentRepository when controller viewstud is used?